examples/bee.js

```diff
@@ -1,16 +1,23 @@
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
   console.log('Usage : node bee.js <host> <port> [<name>] [<password>]')
```

Comment:
I think we should rename the test file though... Bees are now part of the game

Reply:
What should we change the file to? `fly_around`

Reply:
@TheDudeFromCI @rom1504 @Karang

Reply:
Yeah, that works.